GameData/Data.cs

```diff
@@ -20,12 +21,17 @@ public List<Character> CreateAllGameCharacters()
 
         public string GetGameIntroduction()
         {
-            return "Once, there were three ancient instruments." + 
-                "The Harp, Piano, and the Drum." + 
-                "Inside of each instrument there was a magical gem." +
-                "A ruby, sapphire, and a diamond." +
-                "When the gems are merged, it will the create an ancient weapon." +
-                "The Moonrake."; 
+            return "Once, there were three ancient instruments." +
+                " The Harp, Piano, and the Drum." +
+                " Inside of each instrument there was a magical gem." +
+                " A ruby, sapphire, and a diamond." +
+                " When the gems are merged, it will the create an ancient weapon;" +
+                " The Moonrake." +
+                "                                                                                                                            Hello, wellcome to Moonrake, a text adventure game. " + 
```

Comment:
If text isn't being displayed correctly then we should upgrade the engine to make sure it's showing up on the screen right instead of adding a bunch of spaces like this.